editoast/src/views/infra/objects.rs

```diff
@@ -155,6 +158,56 @@ mod tests {
         assert_eq!(call_service(&app, req).await.status(), StatusCode::OK);
     }
 
+    #[rstest]
+    async fn get_objects_should_return_switch() {
+        // GIVEN
+        let app = create_test_service().await;
+        let empty_infra = empty_infra(db_pool()).await;
+        let empty_infra_id = empty_infra.id();
+
+        let switch = Switch {
+            id: "switch_001".into(),
+            switch_type: "switch_type_001".into(),
+            ..Default::default()
+        }
+        .into();
+
+        let create_operation = Operation::Create(Box::new(switch));
+        let request = TestRequest::post()
+            .uri(format!("/infra/{empty_infra_id}/").as_str())
+            .set_json(json!([create_operation]))
+            .to_request();
+        let response = call_service(&app, request).await;
+        assert_eq!(response.status(), StatusCode::OK);
+
+        let expected_switch_object = vec![ObjectQueryable {
+            obj_id: "switch_001".into(),
+            railjson: json!({
+                "extensions": {
+                    "sncf": JsonValue::Null
+                },
+                "group_change_delay": 0.0,
+                "id": "switch_001",
+                "ports": {},
+                "switch_type": "switch_type_001"
+            }),
+            geographic: None,
+            schematic: None,
+        }];
```

Comment:
minor:
this should be moved close to its usage IMO (in the `THEN` part).

also minor:
reusing defined strings for reuse/maintenance could be nicer (some other occurrences in tests may also be changed if agreed)
```suggestion
        let expected_switch_object = vec![ObjectQueryable {
            obj_id: switch.get_id().into(),
            railjson: json!({
                "extensions": {
                    "sncf": JsonValue::Null
                },
                "group_change_delay": 0.0,
                "id": switch.get_id(),
                "ports": {},
                "switch_type": switch.switch_type
            }),
            geographic: None,
            schematic: None,
        }];
```